scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -115,17 +137,27 @@ def check_type(var, vtype, msg):
     if not bool(var):
         raise MotrError(errno.EINVAL, f"Empty {msg}.")
 
-def get_machine_id(self):
-    cmd = "cat /etc/machine-id"
-    machine_id = execute_command(self, cmd)
-    machine_id = machine_id[0].split('\n')[0]
-    check_type(machine_id, str, "machine-id")
-    return machine_id
+def configure_machine_id(self):
+    if Conf.machine_id:
+        print("Atul on 140.....")
+        self.machine_id = Conf.machine_id
```

Comment:
Conf store have 4 machine-ids ( 3 storage-node , 1 control node) for 3 node cluster.. whether this will fetch appropriate machine ID from conf store?